nebari-dev/nebari
# Upgrade conda-store to 2024.11.2

## Reference Issues or PRs
Closes #2523 
Closes #2543 



## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR?

Deploy Nebari off of this branch and make sure to use the jupyterlab docker image built from https://github.com/nebari-dev/nebari/pull/2815#issue-2628727966. You can use the following tag:
```yaml
default_images:
  jupyterhub: quay.io/nebari/nebari-jupyterhub:bump-conda-store
  jupyterlab: quay.io/nebari/nebari-jupyterlab:bump-conda-store
```

Furthermore, if you're deploying on an ARM architecture (e.g. M1 mac), you'll need to specify another conda-store image tag (as there's not one tagged with 2024.11.2). This is the corresponding ARM image for that version:
```yaml
conda_store:
  image: quay.io/quansight/conda-store-server
  image_tag: sha-f8875ca
```


Then, follow the conda-store related steps in our testing checklist: https://github.com/nebari-dev/nebari/blob/9b1310b33e89c2c11c3b39128ec792ca80342486/.github/ISSUE_TEMPLATE/testing-checklist.md?plain=1#L23-L25

Furthermore, make sure the JupyterLab conda-store extension is working.





## Any other comments?
